paddle/fluid/framework/threadpool.h

```diff
@@ -135,6 +135,17 @@ class ThreadPool {
   std::condition_variable completed_;
 };
 
+class MultiStreamThreadPool : ThreadPool {
```

Comment:
Maybe the name `MultiStream` would be mixed up with `cudaStream`?

Reply:
Done.